Blish HUD/GameServices/Input/MouseEventArgs.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+
+namespace Blish_HUD.Input {
+    public class MouseEventArgs : EventArgs {
+        public MouseState MouseState { get; }
+
+        /// <summary>
+        /// The relative mouse position when the event was fired.
+        /// </summary>
+        public Point MousePosition { get; }
```

Comment:
when does this get set?

Reply:
This is planned for the future to avoid recalculating it in every consuming event.  I can have it throw NotImplementedException in the future, but that is not the focus of this PR.

Reply:
> almost all files include usings that they dont need. you could remove them :)

I don't think this is enough to hold up the PR, but I will make the changes when convenient.  Generally I would just do this when doing a quick automated cleanup with ReSharper to save time.

Reply:
thats why its not a requested change. i just hit ctrl+alt+f before is leave a file for a commit. it fixes lots of small things

Reply:
> i found out that you can just remove the `&resolved=false&types=BUG` and they will show up again

Oh, that makes sense and is good to know!

Reply:
> thats why its not a requested change. i just hit ctrl+alt+f before i leave a file for a commit. it fixes lots of small things

And oh!  I did not realize.  Well thank you for the feedback.  I'll get that stuff cleaned up in the near future - especially since they are smaller classes that won't change much.